css-layout-classes/style.css

```diff
@@ -0,0 +1,21 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-third,
+.column-one-fourth > img {
+  width: 100%;
+}
+
+* {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+p {
+  font-family: sans-serif, serif;
+}
```

Comment:
You don't have any styling for any of your column classes, please create rules for each of them with the appropriate styling.